main/src/com/google/refine/util/PatternSyntaxExceptionParser.java

```diff
@@ -121,6 +121,7 @@ public String getUserMessage() {
                 // If no special handling in place fall back on error msg
                 // created by java.util.regex.PatternSyntaxException
                 sb.append(exception.getMessage());
```

Comment:
Not sure the StringBuffer is necessary here. It is one time assign. String should be more efficient and straightforward. You can also consider a HashMap.